content/en/bits_ai/mcp_server/setup/_index.md

```diff
@@ -0,0 +1,331 @@
+
+1. Install the MCP Server binary:
+    * macOS and Linux:
+      ```bash
+      curl -sSL https://coterm.datadoghq.com/mcp-cli/install.sh | bash
+      ```
+
+      This installs the MCP Server binary to `~/.local/bin/datadog_mcp_cli` and then you can use it like any other stdio MCP server.<br/><br/>
+
+    * Windows: Download the [Windows version][1].
+
+2. Run `datadog_mcp_cli login` manually to walk through the OAuth login flow.  
+
+    The MCP Server automatically starts the OAuth flow if a client needs it, but doing it manually lets you choose a [Datadog site][2] and avoid the AI client timing out.
+
+3. Configure your AI client to use the Datadog MCP Server. Follow your client's configuration instructions, as MCP Server setup varies between third-party AI clients.
+
+    For example, for Claude Code, add this to `~/.claude.json`, making sure to replace `<USERNAME>` in the command path:
+
+      ```json
+      {
+        "mcpServers": {
+          "datadog": {
+            "type": "stdio",
+            "command": "/Users/<USERNAME>/.local/bin/datadog_mcp_cli",
+            "args": [],
+            "env": {}
+          }
+        }
+      }
+      ```
+
+    Alternatively, you can also configure Claude Code by running the following:
+      ```bash
+      claude mcp add datadog --scope user -- ~/.local/bin/datadog_mcp_cli
+      ```
+
+[1]: https://coterm.datadoghq.com/mcp-cli/datadog_mcp_cli.exe
+[2]: /getting_started/site/
+{{% /tab %}}
+{{< /tabs >}}
+
+### Authentication
+
+The MCP Server uses OAuth 2.0 for [authentication][8]. If you cannot go through the OAuth flow (for example, on a server), you can provide a Datadog [API key and application key][9] as variables in the header of the MCP Server configuration.
+
+{{< code-block lang="json" >}}
+{
+  "mcpServers": {
+    "datadog": {
+      "type": "http",
+      "url": "https://mcp.datadoghq.com/api/unstable/mcp-server/mcp",
+      "headers": { 
+          "DD_API_KEY": "<your-api-key>", 
+          "DD_APPLICATION_KEY": "<your-application-key>"
+      }
+    }
+  }
+}
+{{< /code-block >}}
+
+### Test access to the MCP Server
+
+1. Install the [MCP inspector](https://github.com/modelcontextprotocol/inspector), a developer tool for testing and debugging MCP servers.
+
+    ```bash
+    npx @modelcontextprotocol/inspector
+    ```
+2. In the inspector's web UI, for **Transport Type**, select **Streamable HTTP**.
+3. For **URL**, enter the [MCP Server URL](#connect-to-the-datadog-mcp-server) for your regional Datadog site.
+4. Click **Connect**, then go to **Tools** > **List Tools**.
+5. Check if the [available tools](#available-tools) appear.
+
+## Client compatibility
+
+The following AI clients are compatible with the Datadog MCP Server.
+
+<div class="alert alert-info">Datadog MCP Server is under significant development, and additional supported clients may become available.</div>
```

Comment:
I like the phrasing you chose for the callout text! Tells people that more clients may be coming, but doesn't promise anything. Is in present tense, and is unlikely to become outdated.